stock_warehouse_flow_delivery_refresh/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Stock Warehouse Flow Delivery Refresh",
+    "summary": "Allow to refresh delivery flow when carrier changes",
+    "version": "14.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/wms",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "stock_warehouse_flow",
+        "stock_available_to_promise_release",
+    ],
```

Comment:
as you depend on alpha module stock_warehouse_flow
```suggestion
    ],
    "development_status": "Alpha",
```